Makefile

```diff
@@ -29,13 +29,23 @@ install-tools:
 	# Install linting tools
 	go get -u -v github.com/golang/lint/...
 	go get -u -v github.com/kisielk/errcheck/...
+	go get -u -v github.com/GeertJohan/fgt
```

Comment:
Let's just drop the "lint" changes and do not call the link Make target in the appveyor.yml file. The linting task is already handled perfectly in the Linux runners, so the Windows runner would not find any new problems. Let's keep the Windows support to a minimum.